### PR TITLE
a lot

### PR DESCRIPTION
**Pull request by pyrosliced**

### CODE REVIEW

**Comment by pyrosliced** [comment]
need to fix smth, cant find "draft" button

File: options/Addons/Mixed/Ghoul.md
Comment (0xDrMoe) [inline options/Addons/Mixed/Ghoul.md:1]: next needs to be fixed

File: options/Addons/Mixed/Gravestone.md
Comment (0xDrMoe) [inline options/Addons/Mixed/Gravestone.md:1]: Previous needs to be fixed

File: options/Settings/System.md
Comment (0xDrMoe) [inline options/Settings/System.md:1]: add your name

**Review by 0xDrMoe — CHANGES_REQUESTED** [review]

**Pull request merged**